chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -26,6 +26,7 @@
       <import_gpg_key config:type="boolean">true</import_gpg_key>
     </signature-handling>
     <storage/>
+    <self_update_url><%= @self_update_url %></self_update_url>
```

Comment:
Didn't we want to add some `unless @self_update_url.nil? || @self_update_url.empty?`